nodejs/scripts/get_terms.js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+import { ArgumentParser } from 'argparse';
+
+import { AccessToken } from '../src/access_token.js';
+import { ApiConfig } from '../src/api_config.js';
+import { common_arguments } from '../src/arguments.js';
+import { Scope } from '../src/oauth_scope.js';
+import { Terms } from '../src/terms.js';
+
+/**
+ * This script shows how to use the Terms endpoint to
+ * view related and suggested terms for ads targeting.
+ */
+async function main(argv) {
+  const parser = new ArgumentParser({
+    description: 'Get Related or Suggested Terms'
+  });
+  parser.add_argument('terms', { help: 'Comma-separated list of terms' });
+  parser.add_argument('-r', '--related', { help: 'Get related terms', action: 'store_true' });
+  parser.add_argument('-s', '--suggested', { help: 'Get suggested terms', action: 'store_true' });
+  // limit has to be n because -l is already used for --log-level
+  parser.add_argument('-n', '--limit', { help: 'Limit the number of suggested terms' });
+  common_arguments(parser);
+  const args = parser.parse_args(argv);
+
+  // exactly one of --related or --suggested must be specified
+  if (!args.related && !args.suggested) {
+    console.log('Please specify --related or --suggested');
+    process.exit(1);
```

Comment:
Since you are using parser.error() above, let's use it here too 🤖🤖

Reply:
oops. thanks! not sure how that one got by. :)